SEED-platform/seed
# Added the ability to create multiple compliance programs

#### Any background context you want to provide?
Previously there was only ever one compliance program available from the GUI. Additionally, you weren't able to associated a compliance program with a filter group.

#### What's this PR do?
This PR adds the ability to create, edit, and delete compliance programs from the Program Setup page. Additionally, you can associate a filter group with a compliance program. Only one filter group can be selected for each compliance program.

#### How should this be manually tested?
Go to Program Setup page in Organization Settings and create, edit, and delete a compliance program. Ensure to try associating a filter group with it also.

Note that I ended up merging a few compliance metric related migrations. In hindsight I realized that there was a non-compliance metric migration that depended on one of the migrations that I removed. Therefore, I revised that migration as well. However, you might need to start fresh and migrate all again.

#### What are the relevant tickets?
#3497 

#### Screenshots (if appropriate)
